brewblox_tilt/broadcaster.py

```diff
@@ -74,19 +78,27 @@ def apply_pi4_hack(self):
             self.scanner._mon.get_hci_version = lambda: HCIVersion.BT_CORE_SPEC_4_2
 
     async def prepare(self):
+        await mqtt.listen(self.app, self.names_topic, self.on_names_change)
+        await mqtt.subscribe(self.app, self.names_topic)
+
         device_id = await self.detect_device_id()
         loop = asyncio.get_running_loop()
         self.scanner = BeaconScanner(
             lambda *args: asyncio.run_coroutine_threadsafe(self.on_event(*args), loop),
             bt_device_id=device_id,
-            device_filter=[IBeaconFilter(uuid=uuid) for uuid in parser.TILT_COLOURS.keys()],
+            device_filter=[
+                IBeaconFilter(uuid=uuid)
+                for uuid in parser.TILT_COLORS.keys()
```

Comment:
`TILT_COLORS` (or the old `TILT_COLOURS`) no longer exists in the `parser` module, resulting in 
`<Broadcaster> error during prepare(): AttributeError(module 'brewblox_tilt.parser' has no attribute 'TILT_COLORS')`
Perhaps you need `const.TILT_UUID_COLORS` ?

Reply:
Seems that slipped by when testing in simulation. Pushed a version with a fix now while I'm writing tests.